docker/environment.yml

```diff
@@ -7,19 +7,22 @@ channels:
   - fortiers  # for fstd2nc
 dependencies:
   # clisops direct dependencies: https://github.com/conda-forge/clisops-feedstock/blob/master/recipe/meta.yaml
+  # figanos direct dependencies: https://github.com/conda-forge/figanos-feedstock/blob/master/recipe/meta.yaml
   # ravenpy direct dependencies: https://github.com/conda-forge/ravenpy-feedstock/blob/master/recipe/meta.yaml
   # xclim direct dependencies: https://github.com/conda-forge/xclim-feedstock/blob/master/recipe/meta.yaml
   # xhydro direct dependencies: https://github.com/conda-forge/xhydro-feedstock/blob/main/recipe/meta.yaml
   # xscen direct dependencies: https://github.com/conda-forge/xscen-feedstock/blob/main/recipe/meta.yaml
   # xsdba direct dependencies: https://github.com/conda-forge/xsdba-feedstock/blob/main/recipe/meta.yaml
 
-  # Pin latest clisops, ravenpy, xclim, xhydro, xscen, and xsdba to avoid downgrading during the second installation phase.
+  # Pin latest clisops, figanos, ravenpy, xclim, xhydro, xscen, and xsdba to avoid undesired downgrades.
```

Comment:
add `undesired downgrades during the second installation phase` so we know when the downgrade happen?

Reply:
That's what I was wondering about as well: When is this second installation phase? We don't have a `pip`-install phase anymore.

Reply:
The first phase: https://github.com/Ouranosinc/PAVICS-e2e-workflow-tests/blob/b8a9fe56aba80354b1dcae7dc43cc3a5656208ec/docker/Dockerfile#L49

The second phase: https://github.com/Ouranosinc/PAVICS-e2e-workflow-tests/blob/b8a9fe56aba80354b1dcae7dc43cc3a5656208ec/docker/Dockerfile#L52

Vague memory we have 2 phases to decrease the memory requirement for the dependency solver and/or to avoid it takes abnormally long to solve all dependencies if everything is done in one shot.

Reply:
Not adding back "during the second installation phase" and then reference to `Dockerfile`?

Reply:
Ah you mentioned the 2 phases in the Dockerfile in the line below.  I wonder how much it is clear for other than me and you that any downgrade is only occurring during the 2nd phase.  Anyways, it's fine, PR already merged.